<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>App_Settings_Advanced</class>
 <widget class="QDialog" name="App_Settings_Advanced">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advanced view settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="edit_view_refresh"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="edit_batch_size"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_view_refresh">
     <property name="text">
      <string>View refresh time (ms):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_batch_size">
     <property name="text">
      <string>Message batch size:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>1 - clear all messages once max messages limit is reached;
2 - clear 50% of all messages (older go first) on reaching the limit;
3 - clear 33%;
4 - 25%,  etc.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="edit_clearing_ratio"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_clearing_ratio">
     <property name="text">
      <string>View clearing ratio:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Sets how many new messages can appear in the view on each refresh.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Controls the frequency of the main window
or history browser views refreshing.</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="button_highlighting">
     <property name="text">
      <string>Configure highlighting settings</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edit_clearing_ratio</tabstop>
  <tabstop>edit_batch_size</tabstop>
  <tabstop>edit_view_refresh</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>App_Settings_Advanced</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>App_Settings_Advanced</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
